Add after_session_confirm hook after session confirmation (#237)
mikker/passwordless: README.md

  config.sign_out_redirect_path = '/' # After a user signs out

  config.paranoid = false # Display email sent notice even when the resource is not found.

>   config.after_session_confirm = ->(session) {} # Called after a session is confirmed.

Reviewer: ```suggestion
  config.after_session_confirm = ->(request, session) {} # Called after a session is confirmed.
```

Let's just always send req + session in this. The other one is mostly there to keep the api backwards compatible.

end
```

> ## After Session Confirmation Hook
> The after_session_confirm hook is called automatically after a successful session confirmation. It receives the request and session objects as arguments, which you can use to access the authenticated user (via session.authenticatable) and perform any necessary operations.
> 
> You can configure the after_session_confirm hook in your Passwordless configuration:
> 
> ```ruby
> Passwordless.configure do |config|
>   # ... other configuration options ...
> 
>   config.after_session_confirm = ->(session, request) {
>     # Your custom logic here
>     user = session.authenticatable
>     user.update(email_verified: true)
>     user.update(last_login_ip: request.remote_ip)
>   }
> end
> ```
> 

Reviewer: ```suggestion
## After Session Confirm Hook

An `after_session_confirm` hook is called after a successful session confirmation – in other words: after a user signs in successfully.

```ruby
Passwordless.configure do |config|
  config.after_session_confirm = ->(session, request) {
    user = session.authenticatable
    user.update!(
      email_verified: true.
      last_login_ip: request.remote_ip
    )
  }
end
```

```
